static/app/stories/view/storySidebar.tsx

```diff
@@ -9,9 +9,14 @@ import {useStoryBookFiles} from './useStoriesLoader';
 
 export function StorySidebar() {
   const {foundations, typography, layout, core, shared} = useStoryBookFilesByCategory();
+  function scrollIntoView(node: HTMLElement | null) {
+    node
+      ?.querySelector('[aria-current="page"]')
+      ?.scrollIntoView({behavior: 'instant', block: 'nearest'});
+  }
```

Comment:
this needs to be moved outside of the component please to become referentially stable. Otherwise, react will all this on every render.